pkg/deploy/deploy_rp.go

```diff
@@ -30,7 +31,7 @@ func (d *deployer) DeployRP(ctx context.Context) error {
 
 	deploymentName := "rp-production-" + d.version
 
-	b, err := Asset(generator.FileRPProduction)
+	b, err := staticassets.EmbeddedFiles.ReadFile(generator.FileRPProduction)
```

Comment:
could you rename the variable at the same time ?

Reply:
Yes, I'll get this updated as well.